README.md

```diff
@@ -26,4 +24,21 @@ The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serv
 - Integrated 3rd party 2D and 3D physics engine
 - Procedural terrain and world generation
 - Artificial Intelligence
-- Audio system
\ No newline at end of file
+- Audio system
+
+
+## Short term goals :
+*Note: this is subject to change at any time! Follow the roadmap over at [hazelengine.com/roadmap](http://hazelengine.com/roadmap).*
+By the end 2020, we want to make a game using the Hazel game engine. Not like the time I made a game in one hour using the engine, but this time by using the proper tools that would be required to make a game with Hazel. This means we need to add a full 2D workflow:
```

Comment:
In my last suggestion, I did mention you should put a new line inbetween the note and the text of the paragraph, as of now, it concatenates both after each other, as shown in the figure below.
![image](https://user-images.githubusercontent.com/26593477/93741770-4ca62800-fbed-11ea-8e2e-82d206b03188.png)

```suggestion
## Short term goals :
*Note: this is subject to change at any time! Follow the roadmap over at [hazelengine.com/roadmap](http://hazelengine.com/roadmap).*

By the end 2020, we want to make a game using the Hazel game engine. Not like the time I made a game in one hour using the engine, but this time by using the proper tools that would be required to make a game with Hazel. This means we need to add a full 2D workflow:
```

Reply:
Fixed